src/main/webapp/login-script.js

```diff
@@ -1,25 +0,0 @@
-function onSignIn(googleUser) {
```

Comment:
Same question on all these scripts. Your edits may be the latest modification on these so if you delete it it's going to be deleted in master.